src/main/java/htsjdk/variant/vcf/VCFHeaderLineTranslator.java

```diff
@@ -65,10 +73,23 @@
     /**
      * parse a VCF4 line
      * @param valueLine the line
+     * @param expectedTagOrder List of expected tags
      * @return a mapping of the tags parsed out
      */
     @Override
     public Map<String, String> parseLine(String valueLine, List<String> expectedTagOrder) {
+        return parseLine(valueLine, expectedTagOrder, Collections.emptySet());
+    }
+    
+    /**
+     * parse a VCF4 line
+     * @param valueLine the line
+     * @param expectedTagOrder List of expected tags
+     * @param optionalTags Set of tags that may or may not be present.
+     * @return a mapping of the tags parsed out
```

Comment:
The interface should have this javadoc, so it's present on all subclass implementations.

Reply:
Will do